specviz/ui/widgets/sub_windows.py

```diff
@@ -183,16 +205,33 @@ def get_roi_mask(self, layer=None, container=None, roi=None):
 
         return mask
 
-    def add_roi(self, *args, **kwargs):
-        view_range = self._plot_item.viewRange()
-        x_len = (view_range[0][1] - view_range[0][0]) * 0.5
-        x_pos = x_len * 0.5 + view_range[0][0]
+    def eventFilter(self, widget, event):
+        if (event.type() == QEvent.KeyPress):
+            key = event.key()
+
+            if key == Qt.Key_Delete or key == Qt.Key_Backspace:
+                for roi in self._rois:
+                    if roi.mouseHovering:
+                        roi.sigRemoveRequested.emit(roi)
+
+                return True
+
+        return QWidget.eventFilter(self, widget, event)
+
+    def add_roi(self, bounds=None, *args, **kwargs):
```

Comment:
Something bad is going on here.  I can't add an ROI.  Log has the following error
```
ERROR:root:Exception in 'add_roi':
Traceback (most recent call last):
  File "/Users/mhill/miniconda3/envs/specviz-dev/lib/python3.5/site-packages/specviz-0.2.2rc5.dev666-py3.5.egg/specviz/core/comms.py", line 195, in wrapper
    return func(*args, **kwargs)
  File "/Users/mhill/miniconda3/envs/specviz-dev/lib/python3.5/site-packages/specviz-0.2.2rc5.dev666-py3.5.egg/specviz/ui/widgets/windows.py", line 102, in add_roi
    window.add_roi(bounds=bounds)
  File "/Users/mhill/miniconda3/envs/specviz-dev/lib/python3.5/site-packages/specviz-0.2.2rc5.dev666-py3.5.egg/specviz/ui/widgets/sub_windows.py", line 228, in add_roi
    start, stop = bounds
TypeError: 'bool' object is not iterable
```